pkg/guard/guard.go

```diff
@@ -12,12 +12,13 @@ const fetchLimit = 100
 
 //go:generate mockery --name Guard
 type Guard interface {
-	IsAllowed(request *ladon.Request) error
+	CreatePolicy(pol ladon.Policy) error
+	DeletePolicy(pol ladon.Policy) error
+	Get(id string) (ladon.Policy, error)
```

Comment:
just to make it consistent maybe `GetPolicy()` ?

Reply:
Agree